src/transformers/modeling_utils.py

```diff
@@ -2344,15 +2345,35 @@ def from_pretrained(
         else:
             model_kwargs = kwargs
 
+        # get the quantization method inside the config of the model if it exists
+        quantization_method = None
+        if hasattr(config, "quantization_config"):
+            if config.quantization_config.get("load_in_8bit", False):
+                quantization_method = QuantizationMethod.BITS_AND_BYTES
+            else:
+                quantization_method = config.quantization_config.get("quant_method", None)
```

Comment:
```suggestion
            quantization_method = config.quantization_config.get("quant_method", QuantizationMethod.BITS_AND_BYTES)
```
Still dont'get why it can't just be this.

Reply:
indeed, we should do that ;) I misunderstood the first message